nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -15,12 +15,12 @@ in
     services.buildkite-agent = {
       enable = mkEnableOption "buildkite-agent";
 
-      token = mkOption {
-        type = types.either types.str types.path;
+      tokenPath = mkOption {
```

Comment:
The rename can only be done on master, for 17.09 we'll have to keep backwards compatibility :(

For master use `mkRenamedOptionModule` for renaming